ibis/backends/flink/datatypes.py

```diff
@@ -88,7 +88,9 @@ def from_ibis(cls, dtype: dt.DataType) -> DataType:
         elif dtype.is_time():
             return DataTypes.TIME(nullable=dtype.nullable)
         elif dtype.is_timestamp():
-            return DataTypes.TIMESTAMP(nullable=dtype.nullable)
+            return DataTypes.TIMESTAMP(
+                precision=dtype.scale if dtype.scale else 6, nullable=dtype.nullable
```

Comment:
```suggestion
                precision=dtype.scale if dtype.scale is not None else 6, nullable=dtype.nullable
```

since `dtype.scale` can be `0` which means seconds precision.